cassandrademo/source/c/CMakeLists.txt

```diff
@@ -15,25 +15,23 @@
 #
 
 cmake_minimum_required(VERSION 2.8.8)
+project(C-SDK-sample C)
 
-project (C-SDK-sample)
-enable_language(C)
+find_package(OpenSSL REQUIRED)
+find_package(Threads REQUIRED)
```

Comment:
This error occurs without thread:
/usr/lib/gcc/x86_64-linux-gnu/5/../../../../lib/libwiringPi.so: undefined reference to `pthread_create'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../../lib/libwiringPi.so: undefined reference to `pthread_cancel'
/usr/lib/gcc/x86_64-linux-gnu/5/../../../../lib/libwiringPi.so: undefined reference to `pthread_join'